central/policy/datastore/datastore_impl.go

```diff
@@ -20,11 +21,17 @@ import (
 	searchPkg "github.com/stackrox/rox/pkg/search"
 	"github.com/stackrox/rox/pkg/set"
 	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/uuid"
 )
 
 var (
 	log       = logging.LoggerForModule()
 	policySAC = sac.ForResource(resources.Policy)
+
+	policyCtx = sac.WithGlobalAccessScopeChecker(context.Background(),
```

Comment:
maybe something like allAccessCtx would be more descriptive

Reply:
"descriptive" but "more than required" right? are you wanting to change this to allAccessCtx ?